action.yml

```diff
@@ -1,22 +1,45 @@
-name: 'Composite Run Steps Action Template'
+name: 'Check Jira Deployment Attestations'
 
-description: 'Add description here'
+description: 'This action will check a deployment task for technical, qa, and stakeholder approvals. If one is not found then the action will fail.'
```

Comment:
I think the description should be updated.

Reply:
Definitely! And those other things you mentioned as well!